templates/settings.py.template

```diff
@@ -59,6 +59,8 @@ INSTALLED_APPS = [
     'rest_framework',
```

Comment:
Please, follow already used style of referencing to issue in the changeset description.